packages/react-app/src/components/utils/transaction-progress-component.tsx

```diff
@@ -15,8 +15,8 @@ const WrapperStyled = styled.div`
   flex-direction: column;
 `;
 
-const MessageStyled = styled.div`
-  color: ${({ messageColor }: any) => messageColor};
+const MessageStyled = styled.div<{ messageColor?: string }>`
```

Comment:
That prop should be required or optional?

Reply:
I think required so we can handle default value outside with theme